<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_01">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>종가 예측</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_01">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>문자열 출력란</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>76</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>삼성</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>주식2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>주식3</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="date_02">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_01">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_02">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>계산</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_futures">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>선물</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_world_indices">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>세계지수</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_currencies">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>환율</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
